research/flamby/utils.py

```diff
@@ -98,8 +98,8 @@ def evaluate_model_on_dataset(
             if is_apfl:
                 preds = model(input)["personal"]
             else:
-                preds = model(input)
-            preds = preds if isinstance(preds, dict) else {"predictions": preds}
+                preds = model(input)[0]
+            preds = preds if isinstance(preds, dict) else {"prediction": preds}
```

Comment:
Due to refactor the output of model version is changed so I needed to update this file.